spring-auto-restdocs-core/src/main/java/capital/scalable/restdocs/constraints/MethodParameterConstraintResolver.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2016 the original author or authors.
```

Comment:
Same as on the other PR. We should decide on how to handle the year for new files.